<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="USMUMSDALVE1" failures="1" tests="11" name="tests" time="53.611" errors="0" timestamp="2024-01-05T15:53:46 IST">
  <testcase classname="tests.ScrollDemo" name="navigateTest" time="18.632"/>
  <testcase classname="tests.ScrollDemo" name="scrollDemo" time="23.644"/>
  <testcase classname="tests.testPagesDemo2" name="testLoginLink" time="0.551"/>
  <testcase classname="tests.testPagesDemo2" name="testRegisterLink" time="0.484"/>
  <testcase classname="tests.testPagesDemo2" name="testRegisterWithValidUsers" time="0.009">
    <skipped/>
  </testcase> <!-- testRegisterWithValidUsers -->
  <testcase classname="tests.testPagesDemo2" name="testRegisterWithValidUsers" time="0.001">
    <skipped/>
  </testcase> <!-- testRegisterWithValidUsers -->
  <testcase classname="tests.testPagesDemo2" name="testRegisterWithValidUsers" time="0.001">
    <skipped/>
  </testcase> <!-- testRegisterWithValidUsers -->
  <testcase classname="tests.testPagesDemo2" name="testRegisterWithValidUsers" time="0.0">
    <skipped/>
  </testcase> <!-- testRegisterWithValidUsers -->
  <testcase classname="tests.testPagesDemo2" name="testRegisterWithValidUsers" time="0.0">
    <skipped/>
  </testcase> <!-- testRegisterWithValidUsers -->
  <testcase classname="tests.testPagesDemo2" name="testRegisterWithValidUsers" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.testPagesDemo2.testRegisterWithValidUsers(testPagesDemo2.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testRegisterWithValidUsers -->
  <testcase classname="tests.testPagesDemo2" name="testLoginWithValidUsers" time="0.002">
    <skipped/>
  </testcase> <!-- testLoginWithValidUsers -->
</testsuite> <!-- tests -->
